custom/icds/management/commands/copy_icds_app.py

```diff
@@ -30,6 +31,9 @@ def handle(self, domain, app_id, version, new_name, **options):
                 form.source = form.source.replace(old_id, new_id)
 
         new_app.save()
+        print("App succesfully copied, you can view it at\n{}".format(
```

Comment:
Nice touch, always appreciate when management commands have a good user experience.

Reply:
I ran this out to test it and got a bunch of deprecation warnings, then nothing.  Figured it needed a little something.